Resources/Prototypes/_Gabystation/Guidebook/sop.yml

```
@@ -52,3 +72,63 @@
   name: guide-entry-pop-marcial
   text: "/ServerInfo/_Gabystation/Guidebook/POPMarcial.xml"
   priority: 6
+
+- type: guideEntry
+  id: POP Multa
+  name: guide-entry-pop-multa
+  text: "/ServerInfo/_Gabystation/Guidebook/POPMulta.xml"
+  priority: 7
+
+# POP Departamental
+
+- type: guideEntry
+  id: POP Comando
+  name: guide-entry-pop-comando
+  text: "/ServerInfo/_Gabystation/Guidebook/POP/Comando/CommandStandardOperatingProcedure.xml"
```

```suggestion
  text: "/ServerInfo/_Gabystation/Guidebook/POP/Command/CommandStandardOperatingProcedure.xml"
```
ou mudar no nome da pasta para `Comando` ou outra coisa e atualizar aqui